src/spelling_wordlist.txt

```diff
@@ -0,0 +1,249 @@
+GScan
+acknowledgement
+aldergrove
+analyse
+aren
```

Comment:
Does it not handle contractions with apostrophes? e.g. `aren't`

Reply:
No, annoyingly.